index.html

```diff
@@ -7345,6 +7426,85 @@ <h5>Presentational Roles Conflict Resolution</h5>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="strong">
+			<rdef>strong</rdef>
+			<div class="role-description">
+				<p>Content which is important, serious, or urgent. See related <rref>emphasis</rref>.</p>
+				<p>The <code>strong</code> role is intended to be used to indicate strong importance, seriousness, or urgency for its contents; not for typographical presentation for presentation's sake. Authors SHOULD use this role only if the absence of the importance would change the meaning of the content.</p>
```

Comment:
For the same reasons I gave for changing the language used to describe the emphasis role, I suggest:

>The purpose of the strong role is to communicate strong importance, seriousness, or urgency. It is not for communicating changes in typographical presentation that are not important to the meaning of the content. Authors SHOULD use the strong role only if its absence would change the meaning of the content.

Reply:
Sounds good to me. Thanks again! Changes made.